types/index.esm.d.ts

```diff
@@ -3033,7 +3033,7 @@ export interface CartesianScaleOptions extends CoreScaleOptions {
      * The label alignment
      * @default 'center'
      */
-    align: Align;
+    align: 'start' | 'center' | 'end' | 'inner';
```

Comment:
Align already has the `'start' | 'center' | 'end'` in it, so would be cleaner if it was just
```suggestion
    align: Align | 'inner';
```

For the rest looks good